pkg/ebpf/initialization/kconfig.go

```diff
@@ -0,0 +1,60 @@
+package initialization
+
+import (
+	"fmt"
+	"github.com/aquasecurity/libbpfgo/helpers"
+	"os"
+)
+
+// Custom KernelConfigOption's to extend kernel_config helper support
+// Add here all kconfig variables used within tracee.bpf.c
+const (
+	CONFIG_ARCH_HAS_SYSCALL_WRAPPER helpers.KernelConfigOption = iota + helpers.CUSTOM_OPTION_START
+	CONFIG_FLATMEM
+	CONFIG_DISCONTMEM
+	CONFIG_SPARSEMEM
+	CONFIG_SPARSEMEM_VMEMMAP
+	CONFIG_SPARSEMEM_EXTREME
+	CONFIG_DYNAMIC_MEMORY_LAYOUT
+	CONFIG_X86_32
+	CONFIG_X86_PAE
```

Comment:
I understand that `helpers.CUSTOM_OPTION_START` allows for expansion of config options, but all of these can be added to the libbpfgo helpers package as well right? When this merges could you create an issue so that we can put them in a more permanent spot?

Reply:
I don't think they should be added to libbpfgo, because they are specific to the future uses I have for them in translating pages to addresses. I don't think any bpf program should have them.
However, you can say that adding them in this PR is out of scope.
In that case I can add the configs in the big PR of my feature solving #1507, but I felt that putting the different parts in different PRs might be easier to review.

Reply:
I agree with @AlonZivony here, that is exactly why the helpers had this option of being extended (because libbpfgo might not need the kconfigs tracee will need). Thanks!